docs/.vuepress/config.js

```diff
@@ -220,6 +232,43 @@ const sidebar = {
             ],
           ],
         },
+        {
+          title: 'Database APIs Reference',
+          collapsable: true,
+          sidebarDepth: 1,
+          children: [
+            ['/developer-docs/latest/developer-resources/database-apis-reference/rest-api.md', 'REST API'],
+            ['/developer-docs/latest/developer-resources/database-apis-reference/graphql-api.md', 'GraphQL API'],
+            {
+              title: 'Query Engine API',
```

Comment:
Not sure if I would put this under dev resources, this might be better under development parent

Reply:
Thanks for the feedback, Derrick!

Maybe the section name ('Developer Resources') does not really fit anymore, but this is still the section where we put all of our APIs and reference docs (CLI, global Strapi API, Content API…)